deploy/helm/kuberhealthy/templates/service.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   annotations:
   {{- if and .Values.prometheus.enabled (not .Values.prometheus.serviceMonitor.enabled) }}
     prometheus.io/scrape: "true"
-    prometheus.io/port: "80"
+    prometheus.io/port: "8080"
```

Comment:
@joshulyne can you verify 8080 is the right port here??

Reply:
yupp, he's right!